### ISSUE TITLE
Anchor wake (and sleep all?) commands

### ISSUE THREAD

**Issue by rntz** [knausj85/knausj_talon#391]
Now that phrase: skip() is enabled by default on w2l this should be significantly better, but there's no use case for un-anchored wake commands that I know of, and I think @knausj85 mentioned feeling that conformer might benefit from anchoring them?

**Comment by pokey** [comment]
I can't come up with a use case for not front-anchoring wake or not back-anchoring sleep, so adding those anchors seem like no-brainers. I could see @rntz 's versions of momentary being an argument for not back-anchoring wake, and potentially issuing some commands followed by a sleep as an argument for not front-anchoring sleep, but those seem like pretty advanced usage. If you're at that point you'd prob just customise it yourself

So long story short yeah I agree these should be anchored on both sides by default

Tho not sure what this has to do with conformer?

**Comment by knausj85** [comment]
@pokey Replace "conformer" with "wav2letter." Conformer's just the first w2l model I've invested significant time into. 

It's probably time to start splitting the commands something like the below. This may be better as a different issue though.

wake_up.talon
```
#defines the various commands to wake/sleep talon
mode: all
-
^welcome back$:
    user.mouse_wake()
    user.history_enable()
    user.talon_mode()
^sleep all$:
    user.switcher_hide_running()
    user.history_disable()
    user.homophones_hide()
    user.help_hide()
    user.mouse_sleep()
    speech.disable()
    user.engine_sleep()
^talon sleep$: speech.disable()
^talon wake$: speech.enable()
```

wake_up_dragon.talon:
```
mode: all
speech.englne: dragon
-
# begin: these commands are really for windows & mac with Dragon.
^dragon mode$: user.dragon_mode()
^talon mode$: user.talon_mode()
```

modes.talon
```
not mode: sleep
-
^dictation mode$:
    mode.disable("sleep")
    mode.disable("command")
    mode.enable("dictation")
    user.code_clear_language_mode()
    mode.disable("user.gdb")
^command mode$:
    mode.disable("sleep")
    mode.disable("dictation")
    mode.enable("command")
```

**Comment by pokey** [comment]
LGTM!
